WalletWasabi.Fluent/Controls/NavBarItem.axaml.cs

```diff
@@ -74,17 +60,26 @@ protected override void OnPropertyChanged<T>(AvaloniaPropertyChangedEventArgs<T>
 			UpdateIndicatorOrientationPseudoClasses(change.NewValue.GetValueOrDefault<Orientation>());
 		}
 
-		if (change.Property == IsSelectableProperty)
+		if (change.Property == IsSelectedProperty)
 		{
-			UpdatePseudoClass(":selectable", change.NewValue.GetValueOrDefault<bool>());
+			UpdatePseudoClass(":selected", change.NewValue.GetValueOrDefault<bool>());
 		}
+	}
 
-		if (change.Property == IsSelectedProperty)
+	protected override void OnPointerPressed(PointerPressedEventArgs e)
+	{
+		base.OnPointerPressed(e);
+
+		if (Command != null && Command.CanExecute(default))
```

Comment:
Can be simplified to:

```suggestion
		if (Command?.CanExecute(default) is true)
```

But your option is fine too because of being a bit more explicit.

Reply:
It is fine as it is